src/test/java/org/hyperledger/fabric/sdkintegration/End2endIT.java

```diff
@@ -783,7 +783,7 @@ policy OR(Org1MSP.member, Org2MSP.member) meaning 1 signature from someone in ei
                     }
 
                 }
-                assertEquals(numberEventsExpected, chaincodeEvents.size());
+                assertTrue("Chaincode events less than expected", chaincodeEvents.size() >= numberEventsExpected);
```

Comment:
An error message along the lines of `"Expected at least " + numberEventsExpected + " chaincode events, got " + chaincodeEvents.size()` would be more informative

Reply:
Changed.